npm/build-docs.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
```

Comment:
Since all scripts are being run via node, is this line required? Doubt this will make much of a difference in windows as well.

We can remove this line, and the associated test.